### ISSUE TITLE
Nextjs Image layout="fill" and TailwindCSS take full page

### ISSUE THREAD

**Issue by StErMi** [vercel/next.js#19471]
# Bug report

## Describe the bug

I'm trying to add Nextjs Image to my current project but it seems to ignore the Tailwind classes.
This is the example code: `<Image className="object-cover w-full h-48" layout="fill" src={business.image} alt={business.name} />`

When I display the image it totally ignores the w/h restriction of the CSS and it will be displayed in full size taking the whole page.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

Se above

## Expected behavior

The expected behavior would be to respect the w/h expressed in the CSS

## System information

- OS: macOS
- Browser (if applies) chrome
- Version of Next.js: 10.0.1
- Version of Node.js: v12.18.1
- Deployment: local

## Additional context

Nothing

**Comment by GilesC9** [comment]
I came across this as well, adding relative to the container div fix it for me.

```         
 <div className="w-64 h-64 relative">
            <Image
              layout="fill"
              objectFit="cover"
              quality={100}
              src={getStrapiMedia(x.bannerImage.formats?.large ?? x.bannerImage)}
              alt={x.bannerImage.alternativeText}
            />
</div>
```

**Comment by RyannosaurusRex** [comment]
Same. @GilesC9's suggestion does sort of work. I'm still required to specify a specific height and width in addition to relative, though`w-64 h-auto relative` doesn't work, for example.  

It seems there are some weird cases where this is broken, and I'm unable to do what I *actually* want, which is to make an image be a specific width but allow the height to be responsive based on the image's actual dimensions.

**Comment by StErMi** [comment]
> I came across this as well, adding relative to the container div fix it for me.
> 
> ```
>  <div className="w-64 h-64 relative">
>             <Image
>               layout="fill"
>               objectFit="cover"
>               quality={100}
>               src={getStrapiMedia(x.bannerImage.formats?.large ?? x.bannerImage)}
>               alt={x.bannerImage.alternativeText}
>             />
> </div>
> ```

yeah `relative` to the parent container works but it can mess with your current layout (like my scenario) that's why I'm not willing to take it as a solution.

**Comment by lachlanjc** [comment]
Yep, this is not Tailwind-specific. You unfortunately can't directly style `Image` like that, with Tailwind, inline styles, or other CSS-in-JS solutions. Wrapper elements are definitely your best bet.

**Comment by Vincent-Carrier** [comment]
I feel like there should be a prop to turn off the `padding-bottom` div. It really messes with layouts in a way that a typical `<img />` doesn't.

**Comment by lachlanjc** [comment]
I'd love that—definitely the main difficult in switching to Next Image Optimization for me. There's still a whole bunch of layout bugs on my sites from the divs, & I had to make a bunch of CSS changes already.

**Comment by kawinie** [comment]
Try putting `sizes="100%"` makes everything work properly.

**Comment by styfle** [comment]
Closing since [the answer](https://github.com/vercel/next.js/issues/19471#issuecomment-737433092) above is the same as the [`layout`](https://nextjs.org/docs/api-reference/next/image#layout) documentation.

![image](https://user-images.githubusercontent.com/229881/150255950-ba0222e3-2278-4d3a-b766-9ed1c33107e1.png)
